dsa-23au/java-dsa/Torsten-David.app/src/main/java/com/ActivityTor/app/DataStorage.java

```diff
@@ -21,6 +21,9 @@ public class DataStorage {
     private SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
 
     // add public getters and setters for BST
+    public void addToSoundTree(double soundLevel, String date){
```

Comment:
Very cool, it looks like you're using a binary search tree, and then recursively traversing all the left children to a leaf (for the minimum value) and all the right children to a leaf (for the maximum value).

Reply:
that is correct